apps/full-site-editing/full-site-editing-plugin/block-helpers/index.ts

```diff
@@ -0,0 +1 @@
+export * from './block-helpers';
```

Comment:
Let's put the helper function right here, the additional indirection doesn't seem helpful.

Reply:
:+1:

Reply:
You're absolutely right, no point in the additional indirection if it's exporting everything from `block-helpers` anyway...